.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -359,29 +364,58 @@
             MauiSplashScreen="@(MauiSplashScreen)"
+            InputsFile="$(_MauiSplashInputsFile)"
         />
-        <PropertyGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True' and '$(TargetPlatformIdentifier)' != 'maccatalyst'">
-            <_MauiStoryboardForPList>$(_MauiIntermediateStoryboard)</_MauiStoryboardForPList>
+        <PropertyGroup Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'">
+            <_MauiIntermediateStoryboard>$(_MauiIntermediateSplashScreen)MauiSplash.storyboard</_MauiIntermediateStoryboard>
+            <_MauiIntermediatePList>$(_MauiIntermediateSplashScreen)MauiInfo.plist</_MauiIntermediatePList>
         </PropertyGroup>
-        <ItemGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True' and '$(TargetPlatformIdentifier)' != 'maccatalyst'">
-            <MauiImage Include="@(MauiSplashScreen)" />
-            <InterfaceDefinition Include="$(_MauiIntermediateStoryboard)" Link="$([System.IO.Path]::GetFileName($(_MauiIntermediateStoryboard)))" />
-            <FileWrites Include="$(_MauiIntermediateStoryboard)" />
+        <CreatePartialInfoPlistTask
+            Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'"
+            IntermediateOutputPath="$(_MauiIntermediateSplashScreen)"
+            PlistName="MauiInfo.plist"
+            Storyboard="$(_MauiIntermediateStoryboard)"
+        />
+        <ItemGroup Condition="'$(_ResizetizerIsiOSSpecificApp)' == 'True'">
+            <_MauiSplashAssets Include="$(_MauiIntermediateSplashScreen)**\*" />
+            <_MauiSplashStoryboard Include="$(_MauiIntermediateStoryboard)" />
+            <_MauiSplashPList Include="$(_MauiIntermediatePList)" />
+            <_MauiSplashImages Include="@(_MauiSplashAssets)" Exclude="@(_MauiSplashStoryboard);@(_MauiSplashPList)" />
+            <!-- Add the launch storyboard file -->
+            <InterfaceDefinition Include="@(_MauiSplashStoryboard)" Link="%(Filename)%(Extension)" />
+            <!-- Add the splash screen plist to the partial plist files -->
+            <PartialAppManifest Include="@(_MauiSplashPList)" />
+            <!-- Add the splash screen images to BundleResource -->
+            <BundleResource Include="@(_MauiSplashImages->'%(FullPath)')">
+                <LogicalName>%(_MauiSplashImages.Filename)%(_MauiSplashImages.Extension)</LogicalName>
+                <TargetPath>%(_MauiSplashImages.Filename)%(_MauiSplashImages.Extension)</TargetPath>
+            </BundleResource>
+        </ItemGroup>
+        <!-- If on Windows, using build host, copy the files over to build server host too -->
+        <ItemGroup Condition="'$(BuildSessionId)' != '' and '$(_ResizetizerIsiOSSpecificApp)' == 'True' and '$(IsMacEnabled)'=='true'">
+            <_MauiSplashToCopyToBuildServer Include="@(_MauiSplashAssets)">
+                <TargetPath>%(Identity)</TargetPath>
+            </_MauiSplashToCopyToBuildServer>
         </ItemGroup>
+        <CopyFilesToBuildServer
```

Comment:
I'm not familiar with these targets and how they interact with the iOS ones, but it shouldn't be necessary to explicitly copy files to the Mac since this should be done by any iOS task  taking the BundleResource item group as input.

That said, we are caching files copied over to the Mac, so if they are being copied twice, we won't send them over again.

Reply:
@mattleibow do you have a `.binlog` of this target in action? It sounds like you shouldn't need to call `<CopyFilesToBuildServer/>` at all?

Reply:
This was originally her, so will merge this and then do some tests and open an issue on macios if it is still needed.

Reply:
Got a separate PR to remove this. Seems to be working. Thanks!